js/interactive-guides/step-content.js

```diff
@@ -233,61 +191,10 @@ var stepContent = (function() {
     if(step.sections){
       for(var i = 0; i < step.sections.length; i++){
         var section = step.sections[i];
-        __updateInstructionsOLD(section);
+        __updateInstructions(section, $stepContent);
       }
     }        
-  };
-
-    // Update the step instruction text
-    var __updateInstructions = function(step, $stepContent) {
-      var stepName = step.name;
-  
-      var index = 0;
-      // Check if any instructions exist for this step
-      if(!contentManager.checkIfInstructionsForStep(stepName)){
-        return;
-      }
-      var lastLoadedInstruction = contentManager.getCurrentInstructionIndex(stepName);
-      // Reached the end of the instructions, so get the last index
-      if(lastLoadedInstruction === -1){
-        lastLoadedInstruction = contentManager.getInstructionsLastIndex(stepName);
-      }
-      do {
-        var instruction = contentManager.getInstructionAtIndex(index, stepName);
-        instruction = __parseInstructionForActionTag(instruction);
-        //console.log("new instruction ", instruction);
-        // Special instruction to track whether the user has completed something but the instruction should not be shown in the DOM.
-        if(instruction && instruction.indexOf("NOSHOW") !== 0){
-          // If the instruction already exists in the page, then show it. Otherwise append it to the bottom of the current content. 
-          var instr = $(".instructionContent[data-step='" + stepName + "'][data-instruction='" + index + "']");
-          if(instr.length > 0){
-            instr.show();
-          }
-          else{
-            instr = __addInstructionTag(stepName, instruction, index);
-            // Check if there is already an instruction for this step, and to append it after that one
-            var stepInstructions = $(".instructionContent[data-step='" + stepName + "']");
-            if(stepInstructions.length > 0){
-              stepInstructions.last().after(instr);
-            }
-            else{
-              $stepContent.append(instr);
-            }          
-          }            
-          
-          contentManager.addCheckmarkToInstruction(stepName, index);        
-        }
-        index++;      
-      } while (index <= lastLoadedInstruction);
-  
-      // Load this step's sections instructions
-      if(step.sections){
-        for(var i = 0; i < step.sections.length; i++){
-          var section = step.sections[i];
-          __updateInstructions(section, $stepContent);
-        }
-      }        
-    };  
```

Comment:
Did you delete the __updateInstructionsOLD or __updateInstructions? Some of my changes for my PR are in step-content.js related to that method. I'm wondering if there will be some conflicts.